### PR TITLE
Explore: Moved 'handles open/close splits in UI and URL' test to the spec folder

### LINKED ISSUE #54775: Explore tests: ExplorePage.test.tsx - move Handles open/close splits in UI and URL test to public/app/features/explore/spec/split.ts
* Move [Handles open/close splits in UI and URL](https://github.com/grafana/grafana/blob/15756ecf86dd3f019cbf9f8356c825f2a76d2647/public/app/features/explore/Wrapper.test.tsx#L157) to a new file explore/spec/split.ts
* Check if we can reuse existing helpers defined in [spec/helper](https://github.com/grafana/grafana/tree/15756ecf86dd3f019cbf9f8356c825f2a76d2647/public/app/features/explore/spec/helper)

### PR DESCRIPTION
**Pull request by harisrozajac**
This was the last task in https://github.com/grafana/grafana/issues/54777.

- It moves the open/close split related logic from `ExplorePage.test.tsx` to a separate file in spec folder.
- Updates test to use standard, non-compact URL format

The remaining thing to do would be to create some unit tests for `ExplorePage.tsx`. The task for that is here: https://github.com/grafana/grafana/issues/54777

Fixes: https://github.com/grafana/grafana/issues/54775

### CODE REVIEW

**Review by gelicia — CHANGES_REQUESTED** [review]
The description for this test still refers solely to the component, when we will want it to reflect it's component-agnostic like the other tests in the `spec` folder.

**Comment by harisrozajac** [comment]
> The description for this test still refers solely to the component, when we will want it to reflect it's component-agnostic like the other tests in the `spec` folder.

@gelicia Ah that makes sense. I wondered about that. I've made the update 👍

File: public/app/features/explore/spec/split.test.tsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+      targets: [{ expr: '{ label="value"}' }],
+    });
+
+    expect(datasources.elastic.query).toBeCalledTimes(1);
+    expect(jest.mocked(datasources.elastic.query).mock.calls[0][0]).toMatchObject({
+      targets: [{ expr: 'error' }],
+    });
+  });
+
+  // TODO: the following tests are using the compact format, we should use the current format instead
```
[the remark this answers is not in the document]
Comment (gelicia) [reply public/app/features/explore/spec/split.test.tsx:142]: For more about the difference between compact and non-compact, see https://docs.google.com/document/d/1YYljgT52u30AbmlKpFjl4s0MC6fZg2SnPIyAAn0aQks/edit#heading=h.5sybau7waq2q 

I feel like this shouldn't take that long and will be easy enough to clean up, but feel free to ignore this.

**Review by gelicia — APPROVED** [review]

File: public/app/features/explore/spec/split.test.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (harisrozajac) [reply public/app/features/explore/spec/split.test.tsx]: @gelicia thank you for the docs, I've updated this to use non-compact format

[the remark this answers is not in the document]
Comment (gelicia) [reply public/app/features/explore/spec/split.test.tsx]: This looks great, and for the other part of this comment, compact tests are handled in their migrator file https://github.com/grafana/grafana/blob/main/public/app/features/explore/hooks/useStateSync/migrators/v0.test.ts#L64 (thanks @Elfo404 !)  so we're all set I think :D

**Pull request merged**